feat: add docs on Apify store tiered discount pricing options (#1572)
apify/apify-docs: sources/platform/actors/running/store.md

always write to [community@apify.com](mailto:community@apify.com).

![Paid Actors' issues tab](./images/store/paid-actors-issues-tab.png)

## Apify Store discounts

> Each Apify subscription plan includes a discount tier (Bronze, Silver, Gold) that provides access to increasingly lower prices on selected Actors.
> Please note that offering these discounts is at the discretion of the Actor owners. Since this is a relatively new feature, not all Actors currently participate in the discount program at the moment.
> Additional discounts are available for Enterprise customers.
> 
> To check an Actor's pricing and available discounts, visit the Pricing section on the Actor's detail page in the Apify Store.
> 
> ![Apify Store discounts](./images/store/apify_store_discounts_web.png)
> 
> In the Apify Console, you can find information about pricing and available discounts in the Actor's header section.
> 
> ![Apify Store discounts](./images/store/apify_store_discounts_console.png)
> 
> ![Apify Store discounts full table](./images/store/apify_store_discounts_full_table.png)

Reviewer: I think Docusaurus admonition works better here. It nicely break up the text and provides visual distinction that highlight the info. I also slightly rewrote the note, I would advise against anchoring such statements in time ( when is `currently`? When is `at the moment`?)

```suggestion
Each Apify subscription plan includes a discount tier (_BRONZE_, _SILVER_, _GOLD_) that provides access to increasingly lower prices on selected Actors.

:::note Discount participation

Discount offers are optional and determined by Actor owners. Not all Actors participate in the discount program.

:::

Additional discounts are available for Enterprise customers.

To check an Actor's pricing and available discounts, visit the Pricing section on the Actor's detail page in the Apify Store.

![Apify Store discounts](./images/store/apify_store_discounts_web.png)

In the Apify Console, you can find information about pricing and available discounts in the Actor's header section.

![Apify Store discounts](./images/store/apify_store_discounts_console.png)

![Apify Store discounts full table](./images/store/apify_store_discounts_full_table.png)
```
